build.gradle

```diff
@@ -8,6 +8,7 @@
 buildscript {
     ext {
         indexApiVersion = "0.2.0"
+        queryApiVersion = "0.3.0"
```

Comment:
✏️ Right now, we only extract version properties that are used in more than one place. Since this is only used once, it can be inlined.